framework/platform/bpl/include/bpl/bpl_cfg.h

```diff
@@ -51,6 +51,13 @@ namespace bpl {
 #define BPL_OPER_MODE_WDS_REPEATER 3
 #define BPL_OPER_MODE_L2NAT_CLIENT 4
 
+/* Platform Management Mode */
+#define BPL_MGMT_MODE_MULTIAP_CONTROLLER_AGENT 0 /* EasyMesh controller and agent */
+#define BPL_MGMT_MODE_MULTIAP_CONTROLLER 1       /* EasyMesh controller */
+#define BPL_MGMT_MODE_MULTIAP_AGENT 2            /* EasyMesh agent */
```

Comment:
I wonder why the macro uses the term "multiap" and the comment us "eazymesh".
I think that is a good opportunity to replace it.

Reply:
I would have done this as part of this change but decided not to since it will require updating prplWrt, UGW and RDKB to the new names, and will break everything until it does.
MultiAP is also correct IMO.
I can fix comment to use Multi-AP terminology.